camel/models/nexa_model.py

```diff
@@ -0,0 +1,173 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+
+import os
+import subprocess
+from typing import Any, Dict, List, Optional, Union
+
+from openai import OpenAI, Stream
+
+from camel.configs import NEXA_API_PARAMS
+from camel.messages import OpenAIMessage
+from camel.types import ChatCompletion, ChatCompletionChunk, ModelType
+from camel.utils import OpenAITokenCounter
+
+
+class NexaModel:
+    """Nexa service interface."""
+
+    def __init__(
+        self,
+        model_type: ModelType,
```

Comment:
there's no defined `ModelType` for Nexa

Reply:
Should I define `ModelType` for Nexa? Or should I just use `str` type for model platforms like Nexa as well as vLLM, ollama?

Reply:
now we support both `ModelType` and `str` as `model_type` input, please update with our latest master branch